packages/jest/src/__test__/dummy.test.ts

```diff
@@ -1,2 +1,7 @@
-// This file is only used to test the config.
-console.log('Hello, world!');
+import { describe, expect, it } from 'vitest';
+
+describe('dummy test file', () => {
```

Comment:
This is not actually run, but needed because otherwise ESLint will fail because there's no tests.